### DISCUSSION TITLE
Тема для обсуждения Rampage Trainer Rus !

### DISCUSSION THREAD

**Discussion by Canek7793** [Canek-77/Rampage-Trainer-Rus#3] [Основная]
## 👋 Добро пожаловать в тему обсуждений !
Мы используем обсуждения как место для общения с другими участниками сообщества. 

Надеемся, что вы будете:
* Взаимодействовать с другими участниками сообщества.
* Задавайте вопросы, которые вас интересуют.
* Делиться идеями.

## Помните, что это сообщество, которое мы строим вместе...

Чтобы начать, прокомментируйте ниже свое предложение либо расскажите о проблемах с русификатором.

**Comment by wfe4wf** [comment]
не устанавливается

**Reply by yefan-p** [reply]
Напиши подробнее, в чем заключается проблема? Вообще в этом архиве только перевод, сам трейнер нужно скачивать отдельно, например [отсюда](https://www.gta5-mods.com/scripts/rampage-trainer).

**Comment by sydzya** [comment]
Большое Вам спасибо!

**Reply by Merwerix** [reply]
Когда выбираю русский язык происходит такой прикол. как исправить?
![image](https://user-images.githubusercontent.com/89612855/131042191-d2716431-d0df-469e-8057-ffc70c96570e.png)
![image](https://user-images.githubusercontent.com/89612855/131042241-4593b4dd-0284-4323-b0e3-b9785b97e8b5.png)

**Reply by yefan-p** [reply]
Сломалась кодировка в файле Russian.xml. Если ты его редактировал, то скачай [Notepad++](https://notepad-plus-plus.org/), и проверь, что установлена кодировка UTF-8. Если установлена не UTF-8, попробуй конвертировать или перекачать файл с переводом (в этом репозитории лежит файл с правильной кодировкой).
![изображение](https://user-images.githubusercontent.com/40020723/131252249-207411b0-6409-4bc2-b3b5-837bc5ef2605.png)

**Reply by Merwerix** [reply]
Спасибо, помогло)

**Comment by Znobed** [comment]
Ребят, помогите. У меня открывается 2 меню (ScriptHook и Rampage) одновременно. Как это исправить?
